github-page/_guides/quick-start-guide.md

```diff
@@ -110,15 +110,19 @@ class KittenClass {
 
 const Kitten = getModelForClass(KittenClass);
 
-let document = await Kitten.create({ name: 'Kitty' });
+let document = new Kitten({ name: 'Kitty' });
```

Comment:
why change this?

Reply:
Because the next line says "has" - I thought that's what it was aiming to demonstrate.

Reply:
i mean `new class` and `await class.create` returns the same document, the only difference is, that `class.create` automatically saves the document already, this is what i would prefer to have in the docs

-> you can read more about it when you search `mongoose create vs new` or something like that